roles/etcd_client_certificates/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - role: etcd_common
+- role: delegated_serial_command
```

Comment:
We probably need to move the majority of the etcd_ca tasks into an included task file as well, which would help with runtimes and still let us have the hard dependency.